src/unix/linux_like/linux/mod.rs

```diff
@@ -1634,6 +1634,34 @@ pub const _SC_XOPEN_STREAMS: ::c_int = 246;
 pub const _SC_THREAD_ROBUST_PRIO_INHERIT: ::c_int = 247;
 pub const _SC_THREAD_ROBUST_PRIO_PROTECT: ::c_int = 248;
 
+pub const _CS_PATH: ::c_int = 0;
```

Comment:
At least some of these should be added to other platforms, e.g. OpenBSD https://man.openbsd.org/confstr.3

Reply:
Agree re additional for other platforms.

A challenge:   The constant is stable and POSIX-defined, but the _constant's value_ is not guaranteed stable across POSIX platforms.